profiles/serializer/normalizer.py

```diff
@@ -44,21 +43,29 @@ def normalize_restricted_profile(profile: Profile) -> dict:
 
 @normalize.register(Affiliation)
 def normalize_affiliation(affiliation: Affiliation) -> dict:
+    if affiliation.department is not None:
+        name = [affiliation.department, affiliation.organisation]
```

Comment:
I wonder if you could build `name` and `components` with the mandatory fields first, and then add the optional values in the right place (0 index and `area` field, respectively). It removes duplication between the `if` branches but it's more complex.

Reply:
good idea. updated.